Userland/DevTools/HackStudio/Language.cpp

```diff
@@ -13,14 +13,18 @@ Language language_from_file_extension(const String& extension)
     VERIFY(!extension.starts_with("."));
     if (extension == "cpp" || extension == "h")
         return Language::Cpp;
-    else if (extension == "js")
+    else if (extension == "js" || extension == "json")
```

Comment:
uh, you sure about that?

Reply:
oh, nvm it was mentioned in commit msg.

Reply:
As I described in the commit message: that's just what TextEditor does.
The current json highlighting works good enough for the json files, but I can see it might not be the best idea.